packages/spine-core-4.2/src/SpineGenerator.ts

```
@@ -0,0 +1,383 @@
+      // - vertexStrideWithTint:    [x, y, z, u, v, r, g, b, a, dr, dg, db] = 12 components
+      //   (Note: we optimized 'da' as uniform instead of buffer attribute)
+      //
+      // Calculation:
+      // - Without tintBlack: 9 - 1 (remove z) = 8 ✓
+      // - With tintBlack:    12 - 1 (remove z) + 1 (add back da) = 12 ✓
+      let vertexSize = tintBlack ? vertexStrideWithTint : vertexStrideWithoutTint - 1;
+      let clippedVertexSize = isClipping ? 2 : vertexSize;
+
+      switch (attachment.constructor) {
+        case RegionAttachment:
+          const regionAttachment = <RegionAttachment>attachment;
```

Biome (2.5.0)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)